snapshot-labs/snapshot
# feat(verify): Add extra data fetching on opening of Verify modal

Fixes #
Add extra data fetching on opening of Verify modal

### Changes 
1. Rename ModalEmailResend.vue to ModalEmailVerify.vue

### How to test
1. Login with an account that is not subscribed to email notifications
2. Subscribe to email notifications, but don't verify email
3. Open Account Menu -> Subscribe -> Modal (it should display Verify Email modal). Close it
4. Verify your email
5. Without refresh open Account Menu -> Subscribe -> Modal (now it should display Email Management modal)

### To-Do

### Self-review checklist
